src/main/java/com/mx/iwaconsolti/calculator/CalculatorApplication.java

```diff
@@ -1,31 +1,20 @@
 package com.mx.iwaconsolti.calculator;
 
-import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-
-import java.util.Scanner;
+import org.springframework.context.ConfigurableApplicationContext;
 
 @SpringBootApplication
-public class CalculatorApplication implements CommandLineRunner {
-
-	public static void main(String[] args) {
-		SpringApplication.run(CalculatorApplication.class, args);
-	}
+public class CalculatorApplication{
 
-	@Override
-	public void run(String... args) throws Exception {
-		Scanner scanner = new Scanner(System.in);
+	public static void main(String[] args){
+		//SpringApplication.run(CalculatorApplication.class, args).close();
 
-		System.out.print("Enter the first number: ");
-		int firstNumber = scanner.nextInt();
+		ConfigurableApplicationContext context = SpringApplication.run(CalculatorApplication.class, args);
+		context.close();
```

Comment:
Can you tell me what are you doing here?

Reply:
* unos ejemplo que encontré en internet lo manejan así, lo probe y funciono:
El código crea y cierra el contexto de aplicación de Spring Boot:
SpringApplication.run() inicia la aplicación y devuelve el contexto.
contexto.close() libera los recursos y cierra el contexto.